### PR TITLE
Added Samurai Fighting Game

### LINKED ISSUE #4631: [New game]: Samurai Fighting Game
### 🎮 Game Request

Game logic and basic description
Screenshot 2024-06-20 at 7 43 14 PM

### Point down the features

Game points

### Select program in which you are contributing

GSSoC24

### Code of Conduct

- [X] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) of this project.

### PR DESCRIPTION
**Pull request by tanyagupta01**
## PR Description 📜
# Samurai Fighting Game

Samurai fighting games have carved out a unique niche in the gaming world, blending intense combat with rich cultural and historical themes. These games often feature stunning visuals, detailed characters, and engaging storylines that immerse players in the world of the samurai. 

## How to Play

- PLAYER 1 can use the following keys:
A D and W to move
spacebar to hit

- PLAYER 2 can use the following keys:
leftarrow rightarrow and uparrow to move
downarrow to hit

Fixes #4631 

- [x] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) & [CODE OF CONDUCT](https://github.com/kunjgit/GameZone/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I have followed proper naming convention showed in [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md)
- [x] I have added screenshot for website preview in assets/images 
- [x] I have added entries for my game in main README.md
- [x] I have added README.md in my folder 
- [x] I have specified the respective issue number for which I have requested the new game.

## Add your screenshots(Optional) 📸
Samurai Fighting Game
Screenshot 2024-06-20 at 10 11 25 PM

### CODE REVIEW

**Comment by kunjgit** [comment]
Thank you @tanyagupta01 ,for creating the PR and contributing to our GameZone 💗 
 Review team will review the PR and will reach out to you soon! 😇 
 Make sure that you have marked all the tasks that you are done with ✅. 
 Thank you for your patience! 😀

File: README.md [hunk 1]
```diff
@@ -794,6 +794,7 @@ This repository also provides one such platforms where contributers come over an
 |[Pottery-Game](https://github.com/kunjgit/GameZone/tree/main/Games/Pottery-Game)|
  | [Ganesh_QR_Maker](https://github.com/kunjgit/GameZone/tree/main/Games/Ganesh_QR_Maker) |
 |[Wheel_of_Fortunes](https://github.com/Saipradyumnagoud/GameZone/tree/main/Games/Wheel_of_Fortunes)|
+|[Samurai_Fighting_Game](https://github.com/Saipradyumnagoud/GameZone/tree/main/Games/Samurai_Fighting_Game)|
```
Comment (ishita-43) [inline README.md:797]: change the format of the link as other games replace your name with the name of this repository

File: README.md [hunk 2]
```diff
@@ -794,6 +794,7 @@ This repository also provides one such platforms where contributers come over an
 |[Pottery-Game](https://github.com/kunjgit/GameZone/tree/main/Games/Pottery-Game)|
  | [Ganesh_QR_Maker](https://github.com/kunjgit/GameZone/tree/main/Games/Ganesh_QR_Maker) |
 |[Wheel_of_Fortunes](https://github.com/Saipradyumnagoud/GameZone/tree/main/Games/Wheel_of_Fortunes)|
```
Comment (ishita-43) [inline README.md:796]: change the name of this also if its your game

**Review by ishita-43 — CHANGES_REQUESTED** [review]
Please make the requested changes

File: README.md (hunk 1 above)
Comment (tanyagupta01) [reply README.md]: @ishita-43 Done.

File: README.md (hunk 2 above)
Comment (ishita-43) [reply README.md:797]: ok also resolve merge conflicts

Comment (tanyagupta01) [reply README.md]: Done

**Review by ishita-43 — APPROVED** [review]
Good to go!!

**Comment by kunjgit** [comment]
Thank you  @tanyagupta01 , for your valuable time and contribution in our GameZone 💗. 
 It’s our GameZone, so Let’s build this GameZone altogether !!🤝
  Hoping to see you soon with another PR again 😇 
 Wishing you all the best for your journey into Open Source🚀

**Pull request merged**